dependencies/kyverno/kustomization.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- https://github.com/kyverno/kyverno/releases/download/v1.11.1/install.yaml
```

Comment:
Do we want to auto update the version?

Reply:
Yes, but I'm not sure we have a working example for getting updates for such references. We have have a bunch of those, but I don't think any of them (e.g. tekton operator) have relevant renovate configs to keep them updated.